### ISSUE TITLE
Missing files

### ISSUE THREAD

**Issue by OrangeSodahub** [hturki/suds#29]
Excellent work! But I wonder where is the `main` function mentioned here:
https://github.com/hturki/suds/blob/9b472ad5e4fd4d810682af984ef65bfaf4d75188/suds/train.py#L22-L24
Unfortunately, I didn't see `script.train` anywhere.

**Comment by OrangeSodahub** [comment]
I found it in `nerfstudio`, may be Line 24 should be `from nerfstudio.scripts.train import main`, maybe I'm wrong

**Comment by six-m** [comment]
> I found it in `nerfstudio`, may be Line 24 should be `from nerfstudio.scripts.train import main`, maybe I'm wrong
@OrangeSodahub 

I think you're wrong.
I meet the same problem as you, and I modify the code follow you, and then I get another issue.
Maybe the correct way is that using the nerfstudio from https://github.com/hturki/nerfstudio/tree/ht/suds instead of the official version.

**Comment by hturki** [comment]
Yeah, unfortunately when we release the code the nerfstudio plugin system wasn't quite ready to be used, and so the current code is fairly tied to the specific version that I developed against (which I think the fork should reflect). At some point I'll try to migrate this code to the official nerfstudio plugin system, but haven't gotten around to it yet...

**Comment by OrangeSodahub** [comment]
@hturki So the codebase cannot work completely (means no other alterities) now until you finish the migration?

**Comment by hturki** [comment]
Have you tried building off my fork (listed in a previous comment)? If that doesn’t work for now, I’ll dig into things some more.

**Comment by hturki** [comment]
closing as stale - let me know if you have any updates!
